Python/Product/TestAdapter/TestAdapter.csproj

```diff
@@ -49,9 +49,13 @@
     <PlatformTarget>AnyCPU</PlatformTarget>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.VisualStudio.TestPlatform.ObjectModel">
+    <!-- <Reference Include="Microsoft.VisualStudio.TestPlatform.ObjectModel">
       <HintPath>$(DevEnvDir)\Extensions\TestPlatform\Microsoft.VisualStudio.TestPlatform.ObjectModel.dll</HintPath>
       <Private>False</Private>
+    </Reference> -->
+    <Reference Include="Microsoft.TestPlatform.ObjectModel">
```

Comment:
Looks like the reference may be missing `.VisualStudio` in its name? (The name of the package is different from the name of the contained binary 😢)...

Also, you may be able to skip the hint path entirely now (similar to other references like MS.VS.Shell.* below which are also referenced via pacakges.config...

Reply:
Thank you for catching this😊. I just made the changes.

Reply:
Btw looks like some test projects below use the old hint path from VS. Would they need similar changes as the current project to avoid build breaks?

Reply:
You are absolutely right, just realized that as well:)